### PR TITLE
Remove hardcode password at data folder

### PR DESCRIPTION
**Pull request by tinawang123**
Remove hardcode password at data folder

- Related ticket: https://progress.opensuse.org/issues/166439
- Verification run:  https://openqa.opensuse.org/tests/4601833
     389ds_server:   http://openqa.opensuse.org/tests/4601832
     pam:   https://openqa.suse.de/tests/15797683#step/pam/33
     xml file update: https://openqa.suse.de/tests/15781372#
    xml.ep file update: https://openqa.suse.de/tests/15781373
    kiwi: https://openqa.suse.de/tests/15797812
    create_hdd_multipath_lvm_encrypt_textmode jobs: https://openqa.suse.de/tests/15807685

### CODE REVIEW

**Review by foursixnine — APPROVED** [review]

**Comment by foursixnine** [comment]
@tinawang123 awesome ^^ thanks!

File: data/mitigation/xen/get_guest_ip.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ expect {
 		exp_continue
         }
         \"*assword\" {
-                send \"nots3cr3t\r\"
+                send \"notXXXX3t\r\"
```
Comment (rfan1) [inline data/mitigation/xen/get_guest_ip.sh:26]: Do you have a test module to handle this change?

Comment (tinawang123) [reply data/mitigation/xen/get_guest_ip.sh]: Updated files,  need verify by other team.

**Review by coolgw — APPROVED** [review]

**Review by jknphy — APPROVED** [review]

**Comment by museQi** [comment]
LGTM

verification:
http://10.200.129.4/tests/79094

**Review by rfan1 — APPROVED** [review]

**Pull request merged**